metrics/configs/build-stats.yaml

```diff
@@ -0,0 +1,48 @@
+metric: build-stats
+description: Calculate the number of builds started and passed each day
+query: |
+  #standardSQL
+  select
+    unix_seconds(day) day,
+    invocations,
+    passes,
+    if (invocations > 0, passes / invocations, 0) rate
```

Comment:
Pass rate should be 1 if there are no invocations.

Reply:
Done